### PR TITLE
LPS-153239 | StructuredContent can be modified priority value

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
StructuredContent can be modified priority value

**Test Plan**
Given a web content is created with default priority
when edit priority value is modified to 3.0
Then the response body includes the modified priority value

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-153239

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro [hunk 1]
```diff
@@ -247,6 +247,14 @@ definition {
 			expected = "${expectedPriorityValueList}");
 	}
 
+	macro assertPriorityFieldWithCorrectValue {
+		var actualPriorityDefaultValue = JSONUtil.getWithJSONPath("${responseToParse}", "$.priority");
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro:251]: `assertPriorityFieldWithCorrectValue` and `assertPriorityFieldWithDefaultValue` do the same thing.
`assertPriorityFieldIsFilteredWithCorrectValue` and `assertPriorityFieldIsSortedWithCorrectValue` do the same thing. 

Let's think about how could we unify these methods

**Review by magjed4289 — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro [hunk 2]
```diff
@@ -231,24 +231,8 @@ definition {
...
-
-		TestUtils.assertEquals(
-			actual = "${actualPriorityValueList}",
-			expected = "${expectedPriorityValueList}");
-	}
-
-	macro assertPriorityFieldWithDefaultValue {
-		var actualPriorityDefaultValue = JSONUtil.getWithJSONPath("${responseToParse}", "$.priority");
+	macro assertPriorityFieldWithCorrectValue {
+		var actualPriorityDefaultValue = JSONUtil.getWithJSONPath("${responseToParse}", "$..priority");
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro:235]: please change to `actualPriorityValue`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/StructuredContentAssetPriority.testcase [hunk 3]
```diff
@@ -49,7 +49,7 @@ definition {
 		}
 
 		task ("Then the response body includes the default priority field") {
-			HeadlessWebcontentAPI.assertPriorityFieldWithDefaultValue(
+			HeadlessWebcontentAPI.assertPriorityFieldWithCorrectValue(
 				expectedPriorityDefaultValue = "0.0",
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/StructuredContentAssetPriority.testcase:53]: please change to `expectedPriorityValue` in all ocurrences

**Review by magjed4289 — CHANGES_REQUESTED** [review]

**Review by magjed4289 — APPROVED** [review]

**Comment by Hijky** [comment]
ci:forward

**Pull request closed without merging**